app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,8 @@
             </intent-filter>
         </activity>
         <activity android:name=".Record" />
-        <activity android:name=".ViewRecording"></activity>
+        <activity android:name=".ViewRecording" />
+        <activity android:name=".activity_record"></activity>
```

Comment:
I dont think this should/needs to be here?

Reply:
the .activity_record one i mean

Reply:
it was from a deleted activity i accidentally made. deleted